test/e2e_node/docker_test.go

```diff
@@ -70,4 +78,170 @@ var _ = framework.KubeDescribe("Docker features [Feature:Docker]", func() {
 			}
 		})
 	})
+
+	Context("when live-restore is enabled [Serial] [Slow] [Disruptive]", func() {
+		It("containers should not be disrupted when the daemon is down", func() {
+			const (
+				podName           = "live-restore-test-pod"
+				containerName     = "live-restore-test-container"
+				volumeName        = "live-restore-test-volume"
+				timestampFilename = "timestamp"
+			)
+
+			isSupported, err := isDockerLiveRestoreSupported()
+			framework.ExpectNoError(err)
+			if !isSupported {
+				framework.Skipf("Docker live-restore is not supported.")
+			}
+
+			By("Check whether live-restore is enabled.")
+			isEnabled, err := isDockerLiveRestoreEnabled()
+			framework.ExpectNoError(err)
+			if !isEnabled {
+				framework.Skipf("Docker live-restore is not enabled.")
+			}
+
+			// Creates a temporary directory that will be mounted into the
+			// container, serving as the communication channel between the host
+			// and the container.
+			By("Create temporary directory for mount.")
+			tempDir, err := ioutil.TempDir("", "")
+			framework.ExpectNoError(err)
+			defer func() {
+				By("Remove temporary directory.")
+				os.RemoveAll(tempDir)
+			}()
+
+			// Creates a container that writes the current timestamp every
+			// second to the timestamp file. We will be able to tell whether
+			// the container is running by checking if the timestamp increases.
```

Comment:
> Why not use "```"?

I wanted to use string concatenation instead of replacement so I didn't use `, but I think either way is fine.

> Why not just launch an nginx, and access it when docker is down? Seems simpler.

Networking may be disrupted. See previous discussion https://github.com/kubernetes/kubernetes/pull/50277#discussion_r131791803.